packages/cloud/src/refreshSpellIconsHandler.ts

```diff
@@ -18,14 +18,18 @@ const gcpCredentials =
 const firestore = new Firestore({
   ignoreUndefinedProperties: true,
   credentials: gcpCredentials,
+  projectId: 'wowarenalogs',
 });
+
 const storage = new GoogleCloudStorage({
   credentials: gcpCredentials,
+  projectId: 'wowarenalogs',
 });
+
 const bucket = storage.bucket('images.wowarenalogs.com');
 
 const MATCH_STUBS_COLLECTION = 'match-stubs-prod';
-const NUMBER_OF_MATCHES = 75;
+const NUMBER_OF_MATCHES = 25;
```

Comment:
still seeing some OOMs running this; it's parsing shuffles fairly regularly so those are x6 for cpu/mem on parsing

Reply:
what does this number control?

Reply:
the number of matches queried per run of the function